build.sbt

```diff
@@ -15,16 +15,16 @@ resolvers += Resolver.bintrayRepo("hmrc", "releases")
 
 lazy val wartremoverSettings =
   Seq(
-    wartremoverErrors in (Compile, compile) ++= Warts.allBut(
+    (Compile / compile / wartremoverErrors) ++= Warts.allBut(
```

Comment:
Use only this 
`.enablePlugins(play.sbt.PlayScala, SbtDistributablesPlugin)
`
and also remove one import

Reply:
Yes same as the above one.